packages/sources/cfbenchmarks-test/src/config/index.ts

```diff
@@ -44,10 +44,9 @@ export const customSettings = {
     description: 'Toggle to set the default endpoint to use WebSockets',
     type: 'boolean',
     required: false,
-    default: true,
     validate: (value?: boolean) => {
       if (!value) {
-        return 'WS_ENABLED must be set to true. Non-WS endpoints are not supported yet.'
+        return 'WS_ENABLED must be set to true'
```

Comment:
> WS_ENABLED must be set to true

Why?

Reply:
I think Jonas means we shouldn't have this validate method anymore. It was only in place to make sure `WS_ENABLED=true` while we didn't have rest.